bot/connector-web/src/main/kotlin/WebBuilders.kt

```diff
@@ -103,11 +103,13 @@ fun <T : Bus<T>> T.webButton(
 fun <T : Bus<T>> T.webUrlButton(
     title: CharSequence,
     url: String,
+    openInSameWindow: Boolean = false,
```

Comment:
Adding the new param after `imageUrl` wouldn't be safer? (for callers, not naming the params)

Reply:
ok